packages/react-day-picker/src/contexts/Modifiers/ModifiersContext.test.tsx

```diff
@@ -0,0 +1,15 @@
+describe('when passing a "selected" custom modifier', () => {
+  test('they should replace the internal "selected" modifiers');
+});
+
+describe('when passing a "disabled" custom modifier', () => {
+  test('they should replace the internal "disabled" modifiers');
```

Comment:
Or perhaps they should be additive with other internal logic related to other props like fromDate, toDate, min, max, etc. I don’t think “replace” is the right word